lookup/lookup.js

```diff
@@ -14,6 +14,7 @@ const request = require('request-promise-native');
 const parse = require('util').promisify(require('csv-parse'));
 const utils = require('../utils.js');
 const config = utils.get_config();
+const transpose = m => m[0].map((x, i) => m.map(x => x[i]));
```

Comment:
I'm confused why this is needed. Can you elaborate on why this is used?

Reply:
It interchanges the rows and columns of a matrix
```
[[responsePromise, usernamesPromise],[responsePromise, usernamesPromise]]
into
[[responsePromise, responsePromise], [usernamesPromise, usernamesPromise]]
```

I was initially calling `map` multiple times, but I think a [transpose](https://devdocs.io/ruby~2.6/array#method-i-transpose) looks cleaner.
```
const responsePromises = promisesMatrix.map(arr => arr[0]);
const usernamesPromises = promisesMatrix.map(arr => arr[1]);
```

Reply:
OK great, thanks! No this is sufficient. This is more about educating me than doubting your approach or asking for you to change it 😄